src/pages/profile/[slug].ts

```diff
@@ -0,0 +1,17 @@
+import { GetServerSideProps } from 'next'
+import { securedProps } from 'middleware/auth/keycloak'
+import ProfilePage from 'components/auth/profile/ProfilePage'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
+  const { slug } = ctx.query
+  return {
+    props: {
+      slug,
```

Comment:
We send the `slug` prop but we never use it in the `ProfilePage` component

There we use it again from `router.query.slug`

Let's leave only one of these

Reply:
Removed passing of slug prop from the server side props in https://github.com/podkrepi-bg/frontend/pull/703/commits/e6c92cdf5b61ceeaa038c5172e7ea6c0165455df